src/webextension/manage/index.js

```diff
@@ -20,7 +20,7 @@ initializeMessagePorts(store);
 
 ReactDOM.render(
   <Provider store={store}>
-    <AppLocalizationProvider baseDir="locales" availableLocales={["en-US"]}
+    <AppLocalizationProvider baseDir="/locales" availableLocales={["en-US"]}
```

Comment:
With `baseDir` defaulting to `"/locales"` in the class definition above, we can remove that prop from here. Then we should also add `bundle="manage"` so our provider knows what `.ftl` file to load.